### ISSUE TITLE
Training Hyperparameters Question

### ISSUE THREAD

**Issue by LISI0037** [felix-ch/f3loc#12]
Hi, Thanks for open-sourcing such a great work! I'm training for 100 epochs, but the results are quite far from what's reported in the paper. Could you provide the exact training settings, including learning rate, optimizer, number of epochs, and any learning rate scheduling used? Want to make sure I'm using the right setup. Thanks.

**Comment by felix-ch** [comment]
Hi, 

we use Adam with a learning rate of 1e-3 for all training, no learning rate scheduling is used. Please refer to https://felix-ch.github.io/f3loc-page/f3loc_supp.pdf 2.Training Details for all details. The augmentation for all training is set to roll and pitch both in [-0.3, 0.3], and the shape loss weight = 20. The batch size for mono is 30, multiview 10, complementary 40. The trainings were done on 4xA100 cluster. 

Best
